patterns/testimonials-review-large-image-right.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Review with large image on right
+ * Slug: twentytwentyfive/testimonials-review-large-image-right
+ * Keywords: testimonial
+ * Categories: testimonials
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:group {"style":{"dimensions":{"minHeight":"100%"},"spacing":{"padding":{"top":"0","bottom":"0","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"blockGap":"var:preset|spacing|60"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","flexWrap":"wrap","verticalAlignment":"space-between"}} -->
```

Comment:
The stack block containing the row and quote should not have left and right padding, they should align with the column edges to align with the rest of the content in the theme and patterns. Like I'm showing in the video:

https://github.com/user-attachments/assets/65024ee3-dd0a-4ffb-b768-f5c9dfad3207

Reply:
Thanks, updated in https://github.com/WordPress/twentytwentyfive/pull/118/commits/6b65a0c983ace36d4f148cd61f8410f4b5ca5744